moveit_core/robot_model/src/robot_model.cpp

```diff
@@ -928,10 +928,16 @@ JointModel* RobotModel::constructJointModel(const urdf::Joint* urdf_joint, const
     {
       if (virtual_joint.child_link_ != child_link->name)
       {
-        RCLCPP_WARN(LOGGER,
-                    "Skipping virtual joint '%s' because its child frame '%s' "
-                    "does not match the URDF frame '%s'",
-                    virtual_joint.name_.c_str(), virtual_joint.child_link_.c_str(), child_link->name.c_str());
+        if (child_link->name == "world" && virtual_joint.type_ == "fixed" && child_link->collision_array.empty() &&
```

Comment:
`child_link->name == "world"`

I thought "world" is the parent link, not the child?